terraform_create_images/modules/loader_vm/scripts/create_and_load_everything_from_scratch.sh

```diff
@@ -55,16 +55,15 @@ load_foreach_parquet() {
 load_json_for_elastic() {
   local data_path=$1
   local index=$2
-  local index_file=$3
```

Comment:
We need the index file parameter.

Reply:
I don't understand. The function doesn't use this parameter. It does the same thing with and without `index_file` (`$3`).